client/platform/desktop/frontend/components/ViewerLoader.vue

```diff
@@ -128,21 +137,21 @@ export default defineComponent({
     </template>
     <template #title-right>
       <RunPipelineMenu
-        :selected-dataset-ids="[id]"
+        :selected-dataset-ids="[modifiedId]"
         :sub-type-list="subTypeList"
         :camera-numbers="camNumbers"
         :running-pipelines="runningPipelines"
         :read-only-mode="readOnlyMode"
         v-bind="{ buttonOptions, menuOptions }"
       />
       <ImportAnnotations
-        :dataset-id="compoundId"
+        :dataset-id="modifiedId"
         v-bind="{ buttonOptions, menuOptions, readOnlyMode }"
         block-on-unsaved
       />
       <Export
         v-if="datasets[id]"
-        :id="compoundId"
+        :id="modifiedId"
```

Comment:
Out of scope, but I think there needs to be a way to export the entire multicamera dataset, not just 1 cam at a time.

Reply:
I think that comes down to implementing on desktop the dataset export and creation of zip files as well.